stubs/tornado/@python2/tornado/locks.pyi

```diff
@@ -11,35 +12,37 @@ class Condition(_TimeoutGarbageCollector):
     def notify_all(self): ...
 
 class Event:
-    def __init__(self): ...
-    def is_set(self): ...
+    def __init__(self) -> None: ...
+    def is_set(self) -> bool: ...
     def set(self): ...
     def clear(self): ...
     def wait(self, timeout: Optional[Any] = ...): ...
 
 class _ReleasingContextManager:
-    def __init__(self, obj): ...
-    def __enter__(self): ...
+    def __init__(self, obj) -> None: ...
+    def __enter__(self) -> None: ...
     def __exit__(self, exc_type, exc_val, exc_tb): ...
 
 class Semaphore(_TimeoutGarbageCollector):
-    def __init__(self, value: int = ...): ...
-    def release(self): ...
+    def __init__(self, value: int = ...) -> None: ...
+    def release(self) -> None: ...
     def acquire(self, timeout: Optional[Any] = ...): ...
-    def __enter__(self): ...
+    def __enter__(self) -> NoReturn: ...
     __exit__: Any
-    def __aenter__(self): ...
-    def __aexit__(self, typ, value, tb): ...
+    def __aenter__(self) -> None: ...
+    def __aexit__(
+        self, typ: Optional[Type[BaseException]], value: Optional[BaseException], tb: Optional[TracebackType]
+    ) -> None: ...
```

Comment:
How can changing these methods help with anything? Python 2 doesn't have `async with` syntax, so aren't `__aenter__` and `__aexit__` just useless in Python 2 stubs?

Reply:
Yes, it really doesn't do much, except make some progress with ensuring that we use `Self` correctly on all `__enter__`/`__aenter__` methods.

Reply:
What do you mean? There is no `Self` in the diff.

Reply:
The `__aenter__` method was unannotated, and I made these changes while verifying that I didn't need to add `Self`.

Reply:
It seems a bit weird to improve annotations for methods that can't be actually used, but whatever.